code/game/jobs/jobs.dm

```diff
@@ -65,12 +65,19 @@ var/list/command_positions = list(
 	"Head of Security",
 	"Chief Engineer",
 	"Research Director",
-	"Chief Medical Officer"
+	"Chief Medical Officer",
+	"Acting Captain",
+	"Acting Head of Personnel",
+	"Acting Head of Security",
+	"Acting Chief Engineer",
+	"Acting Research Director",
+	"Acting Chief Medical Officer"

```

Comment:
Just for general purposes, add abbreviates too, like "Acting HoS", "Acting RD" etc